test/framework/functional/write.go

```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 )
 
-func (f *CollectorFunctionalFramework) WriteMessagesToApplicationLogInNamespace(msg, namespace string, numOfLogs int) error {
+func (f *CollectorFunctionalFramework) WriteMessagesToContainerLogInNamespace(msg, namespace string, numOfLogs int) error {
```

Comment:
Nack this change. In the context of logging "application" has meaning that container does not

Reply:
I need a method to pass in a specific namespace from the infra index ("openshift-authentication-operator"), and realized later that this method did that already.  Since namespace not bound, do you have a suggestion for a common name in order to avoid duplicate methods?

Reply:
update to `WriteMessagesToNamespace`